keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -232,20 +232,40 @@ _This section must be completed when targeting alpha to a release._
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes, but CNIs relying on the new field wont recognize it anymore
+  Yes, but CNIs relying on the new field wont recognize it anymore. Also, 
+  because this is a Feature Gate'd API field, disabling the feature gate and 
+  updating a Network Policy with existing EndPort field will remove this field.
 
 * **What happens if we reenable the feature if it was previously rolled back?**
   Nothing. Just need to check if the data is persisted in `etcd` after the 
-  feature is disabled and reenabled or if the data is missed
+  feature is disabled and reenabled. If the object "NetworkPolicy" is updated
+  during this time window between disabling and re-enabling the EndPort field 
+  from this object might be removed by PrepareForUpdate function.
 
 * **Are there any tests for feature enablement/disablement?**
  
- TBD
+ Yes, and the tests are already being implemented in the REST stragegy package
+ for NetworkPolicy.
 
-### Monitoring Requirements
+ ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
+* **How can a rollout fail? Can it impact already running workloads?**
+  Nothing happens, this is a new Network Policy API Field
+
+* **What specific metrics should inform a rollback?**
+  None
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  TBD / This tests should be done with a defined CNI, so as soon as CNIs starts
+  supporting the new field (by the graduation criteria, at least 4) this should be tested.
```

Comment:
We should probably do manual update tests even without CNI supporting that to see there aren't unexpected bugs.

Reply:
Indeed, can do this weekend

Reply:
I don't want to block the PR on it (especially for Alpha) - just mention that it will be done